packages/office-ui-fabric-react/src/components/Slider/Slider.scss

```diff
@@ -1,24 +1,129 @@
 @import '../../common/common';
 
-.root {
-  user-select: none;
-  margin-bottom: 8px;
-}
+.row, .column {
+  .root {
+    user-select: none;
+  }
+
+  .titleLabel {
+      padding: 0;
+  }
 
-.titleLabel {
-  padding: 0;
+  .line {
+    display: flex;
+    position: relative;
+  }
+
+  .line span {
+    border-radius: 4px;
+    box-sizing: border-box;
+  }
+
+  .slideBox {
+    @include focus-border();
+    background: transparent;
+    border: none;
+    padding: 0;
+    margin: 0;
+    flex-grow: 1;
+    line-height: 28px;
+  }
+
+  .slideBox .thumb {
+    border: 2px solid $ms-color-neutralSecondary;
+    box-sizing: border-box;
+    background: $ms-color-white;
+    display: block;
+    width: 16px;
+    height: 16px;
+    position: absolute;
+    border-radius: 10px;
+  }
+
+  .container {
+    display: flex;
+    flex-wrap: nowrap;
+    align-items: center;
+  }
+
+  .valueLabel {
+    flex-shrink: 1;
+    width: 30px;
+    line-height: 1;
+  }
 }
 
-.line {
-  display: flex;
-  position: relative;
-  width: 100%;
+.row {
+  .root {
```

Comment:
you're nesting here. Can you please avoid?

Reply:
These are your overrides in the specific states on top of the base rules;

I am ok with the nesting as long as it's a few overrides on top of base.